src/inventory/aushada/aushada.jsx

```diff
@@ -52,53 +60,99 @@ const Aushada = (props) => {
   const [stockEmptyResonseMessage, setStockEmptyResonseMessage] = useState(false);
   const [negativeError, setNegativeError] = useState(false);
   const [outwardNumberExists, setOutwardNumberExists] = useState(false);
+  const [inwardNumberExists, setInwardNumberExists] = useState(false);
   const [cookies] = useCookies();
   const { setReloadData } = props;
   const { itemStock} = useItemStockContext();
+  const [isDateSelected, setIsDateSelected] = useState(false);
+  const [selectedDate, setSelectedDate] = useState('');
+  const [enableEaushadhaInwardApi, setEnableEaushadhaInwardApi] = useState('');
+  const [instituteId, setInstituteId] = useState([]);
+  const [instituteIdExists, setInstituteIdExists] = useState(false);
+  const locationUuid = cookies[locationCookieName].uuid;
 
   const { data: stockRoom, error: stockRoomError } = useSWR(
     stockRoomURL(cookies[locationCookieName]?.name.trim()),
     fetcher,
   );
 
+  useEffect(async ()=>{
+    const response=await getRequest(globalPropertyUrl('eAushadha.inward'))
+    if(response==true){
```

Comment:
Here we can also avoid boolean comparision

Reply:
Done